### PR TITLE
added article for this stub

### PR DESCRIPTION
**Pull request by takachou**
added 3 ways for full page background image

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

**Review by thecodingaviator — CHANGES_REQUESTED** [review]
Try to never use personal links, do not keep the html and css blocks apart, put the css in a `<style>` tag, and use the language i.e. html in this case after the three back tick characters. like ```html

File: guide/english/css/tutorials/perfect-full-page-background-image/index.md [hunk 1]
```diff
@@ -2,14 +2,84 @@
...
+.bg-image {
+    background-image: url("myimage.jpg");
+    height: 100%;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+```
+
+see <a href="https://codepen.io/takachou/pen/gBzRwK">codepen for solution 2</a>
```
Comment (cmccormack) [inline guide/english/css/tutorials/perfect-full-page-background-image/index.md]: Please use github markdown style [links](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/docs/style-guide-for-guide-articles.md#links)

File: guide/english/css/tutorials/perfect-full-page-background-image/index.md [hunk 2]
```diff
@@ -2,14 +2,84 @@
...
+```
+html { 
+  background: url(images/bg.jpg) no-repeat center center fixed; 
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+}
+```
+see <a href="https://codepen.io/takachou/pen/bmMRgz">codepen for solution 3</a>
```
Comment (cmccormack) [inline guide/english/css/tutorials/perfect-full-page-background-image/index.md]: Please use github markdown style [links](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/docs/style-guide-for-guide-articles.md#links)

File: guide/english/css/tutorials/perfect-full-page-background-image/index.md [hunk 3]
```diff
@@ -2,14 +2,84 @@
...
 #### More Information:
 <!-- Please add any articles you think might be helpful to read before writing the article -->
```
Comment (cmccormack) [inline guide/english/css/tutorials/perfect-full-page-background-image/index.md]: Can remove comments when filling out a stub

**Review by cmccormack — APPROVED** [review]
These changes look good; conflict resolution will need to be performed and then we can merge.

**Pull request merged**